azkaban-common/src/main/java/azkaban/sla/SlaOption.java

```diff
@@ -30,9 +30,6 @@
 import java.util.Objects;
 import java.util.Set;
 import java.util.stream.Collectors;
-import org.joda.time.DateTime;
```

Comment:
Thanks so much for getting rid of joda! I assume it is used elsewhere as well, but might be good to double check if the import could be removed entirely!